app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1235,7 +1237,12 @@ private void setMapMarkerActions(Marker selectedMarker) {
     @Override
     public void recenterMap(fr.free.nrw.commons.location.LatLng curLatLng) {
         if (curLatLng == null) {
+            showEnableLocationDialog();
             return;
+        } else if (!locationManager.isLocationEnabled()) {
+            // location is not null but it is not enabled. So, user should be prompted
```

Comment:
As we don't handle the NULL case separately anymore, this comment would be misleading and can be removed.

Reply:
Done!